File: DBManager.java
package pe.edu.pucp.softprog.config;
import java.sql.Connection;
import java.sql.DriverManager;
public class DBManager {
    private static DBManager dbManager;
    private String url = "jdbc:mysql://" +
"db-prog3-clases.c8izfhuif1lk.us-east-1.rds.amazonaws.com" + 
            ":3306/" + "prog3";
    private String usuario = "adminprog3";
    private String password = "prog31inf30";
    private Connection con;
    
    public static DBManager getInstance(){
        if(dbManager == null)
            createInstance();
        return dbManager;
    }
    
    private static void createInstance(){
        dbManager = new DBManager();
    }
    
    public Connection getConnection(){
        try{
            Class.forName("com.mysql.cj.jdbc.Driver");
            con = DriverManager.getConnection
        (url, usuario, password);
        }catch(Exception ex){
            System.out.println(ex.getMessage());
        }
        return con;
    }
}

File: DBManager.cs
using MySql.Data.MySqlClient;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedicalSoftDBManager
{
    public class DBManager
    {
        private static DBManager dBManager = new DBManager();
        private string URL = "server=" +
            "labs-1inf30-prog3-20242.cnjqylm6u7zo.us-east-1.rds.amazonaws.com;";
        private string user = "user=" +
            "admin;";
        private string password = "password=" +
            "prog320242labs;";
        private string puerto = "port=" +
            "3306;";
        private string database = "database=" +
            "labPractica;";
        private MySqlConnection con;

        private DBManager()
        {
            String cadena = URL + user + password + puerto + database;
            con = new MySqlConnection(cadena);
        }
        public static DBManager Instance 
        {
            get{
                return dBManager;
            }
        }
        

        public MySqlConnection Connection
        {
            get { 
                return con;
            }
        }

    }
}
